### ISSUE TITLE
Duplicate Imports Rule Not Triggering as Error

### ISSUE THREAD

**Issue by freddiebo** [realm/SwiftLint#5978]
### New Issue Checklist

- [x] I've Updated SwiftLint to the latest version.
- [x] I've searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues).

### Bug Description

The **duplicate_imports** rule in SwiftLint is not being reported as an error even when there are duplicate import statements in the code. This may lead to confusion and clutter in the import section of the files, affecting code readability and maintainability.

Mention the command or other SwiftLint integration method that caused the issue. Include stack traces or command output.

```bash
$ swiftlint lint
```

### Environment

* SwiftLint version: 0.58.0
* Xcode 16.2
* Installation method used Homebrew
* Configuration file:

```yml
opt_in_rules:
  - duplicate_imports

duplicate_imports:
  severity: error
```

**Comment by SimplyDanny** [comment]
Please provide an example where the rule doesn't trigger on.

**Comment by freddiebo** [comment]
The second **import** gives a warning, the build does not fall into an error.

```swift
import UIKit

extension UIWindow {
    static var activeVC: UIViewController? {
        if let controller = UIApplication.appDelegate?.window?.rootViewController {
            if let tabVC = controller as? MainTabBarController,
                let activeVC = tabVC.selectedViewController {
                return activeVC
            } else {
                return controller
            }
        }
        return nil
    }
    
    static var tabBar: MainTabBarController? {
        UIApplication.appDelegate?.window?.rootViewController as? MainTabBarController
    }
}

import UIKit
extension UIWindow {
    var presentedController: UIViewController? {
        rootViewController?.presentedViewController ?? rootViewController
    }
}
```

**Comment by SimplyDanny** [comment]
I see. The severity for this rule is simply not configurable at the moment. That's going to be fixed with #5979.
